### ISSUE TITLE
pwquality should be activated for local users only

### ISSUE THREAD

**Issue by jhrozek** [pbrezina/authselect#24]
The current authselect profiles include pam_pwquality for all users:
```
profiles/sssd/password-auth:password    requisite                                    pam_pwquality.so try_first_pass
profiles/sssd/system-auth:password    requisite                                    pam_pwquality.so try_first_pass
profiles/winbind/password-auth:password    requisite                                    pam_pwquality.so try_first_pass
profiles/winbind/system-auth:password    requisite                                    pam_pwquality.so try_first_pass
```

But we probably should use the local_users_only flag..

**Comment by pbrezina** [comment]
I kept it the same as the current Fedora version. Do you still think we should change it? I know that remote users has usually password policy implemented in LDAP, but tmraz insisted on having it enabled for all users.

**Comment by jhrozek** [comment]
On Wed, Sep 20, 2017 at 11:20:59AM +0000, Pavel Březina wrote:
> I kept it the same as the current Fedora version. Do you still think we should change it? I know that remote users has usually password policy implemented in LDAP, but tmraz insisted on having it enabled for all users.

Are you sure? I just started a Fedora-26 container and the default
passwd-auth contains:

    password    requisite     pam_pwquality.so try_first_pass local_users_only retry=3 authtok_type=
    password    sufficient    pam_unix.so try_first_pass use_authtok nullok sha512 shadow
    password    required      pam_deny.so


after calling "authconfig --enablesssdauth --enablesssd --update" the
file contains:

    password    requisite     pam_pwquality.so try_first_pass local_users_only retry=3 authtok_type=
    password    sufficient    pam_unix.so sha512 shadow nullok try_first_pass use_authtok
    password    sufficient    pam_sss.so use_authtok
    password    required      pam_deny.so

**Comment by pbrezina** [comment]
Fixed with: 09b77f2cc28f66797fe6bb1226b58f392136af97
